public/usage-examples/graphics/draw_triangle/Simple_Red_Triangle-opp.cs

```diff
@@ -0,0 +1,22 @@
+﻿using SplashKitSDK;
+public class SimpleRedTriangle
+{
+    public static void Main()
+    {
+        new SimpleRedTriangle().Run();
+    }
+    public void Run()
+    {
+        Window _window = new Window("Simple Red Triangle", 800, 600);
+
+        while (!_window.CloseRequested)
+        {
+            SplashKit.ProcessEvents();
+            _window.Clear(Color.White);
+            SplashKit.FillTriangle(Color.Red, 150, 50, 250, 250, 50, 250);
+            _window.Refresh();
+        }
+    }
+    
+}
+
```

Comment:
Please review the guide on how to convert from top level to OOP. There should be minimal change to the actual code.

https://thoth-tech.netlify.app/products/splashkit/splashkit-website/tutorials-documentation/04-oop-styling/

Then correct the code.